cavern/src/main/java/org/opencadc/cavern/CavernInitAction.java

```diff
@@ -86,6 +91,21 @@ public CavernInitAction() {
     @Override
     public void doInit() {
         try {
+            // Check config file
+            CavernConfig cavernConfig = new CavernConfig();
+            MultiValuedProperties props = cavernConfig.getConfig();
+
+            // create root directories for node/files
+            String baseDir = props.getFirstPropertyValue(CavernConfig.FILESYSTEM_BASE_DIR);
+            String subPath = props.getFirstPropertyValue(CavernConfig.FILESYSTEM_SUB_PATH);
+            Path root = Paths.get(baseDir + subPath);
```

Comment:
make sure there is a `/` between these?

Reply:
Yes, would be a good idea if the base doesn't end with a / and the subPath doesn't start with one.